packages/host/app/components/operator-mode/code-mode.gts

```diff
@@ -383,9 +384,18 @@ export default class CodeMode extends Component<Signature> {
         return maybeCard;
       }
     }
+    // in order to not get trapped in a glimmer invalidation cycle we need to
+    // unload the card in a different closure
+    this.unloadCard.perform();
     return undefined;
   }
 
+  private unloadCard = task(async () => {
+    await Promise.resolve();
```

Comment:
Is this needed?

Reply:
Yeah, this is what allows us to run this in a different closure. Otherwise we’ll get glimmer invalidation cycles (see comment above).